book/08-customizing-git/sections/attributes.asc

```diff
@@ -298,7 +298,7 @@ Now, when you run `git archive` to create a tarball of your project, that direct
 
 ===== `export-subst`
 
-When exporting files for deployment you can apply `git log`'s formatting and keyword-expansion processing to selected portions of files marked with the `export-subst` attribute.
+When exporting files for deployment you can apply ``gitlog```'s formatting and keyword-expansion processing to selected portions of files marked with the ``export-subst`` attribute.
```

Comment:
Looks like we lost a space here:

```suggestion
When exporting files for deployment you can apply ``git log```'s formatting and keyword-expansion processing to selected portions of files marked with the ``export-subst`` attribute.
```

Reply:
Derp 🙄